File: server/internal/entity/position.go
package entity

import "github.com/gofrs/uuid/v5"

type Position struct {
	ID           uuid.UUID `json:"-"`
	Latitude     float64   `json:"latitude"`
	Longitude    float64   `json:"longitude"`
	MetroStation *string   `json:"metro_station"`
}

File: transformer/db/models/position.py
from peewee import *
from .base_model import BaseModel

import uuid


class Position(BaseModel):
  id = UUIDField(primary_key=True, default=uuid.uuid4)
  latitude = DecimalField(decimal_places=6)
  longitude = DecimalField(decimal_places=6)
  metro_station = CharField(max_length=511, null=True)
